lit.ts

```diff
@@ -258,13 +307,16 @@ export async function mintPKPV2({
 		sendPkpToItself,
 	);
 
-	console.log('config.network:', config.network);
+	console.log("config.network:", config.network);
 
 	const pkpHelper = await getPkpHelperContract();
 	const pkpNft = await getPkpNftContract();
 
 	// first get mint cost
-	const mintCost = await pkpNft.mintCost();
+	const mintCost = await pkpNft.mintCost().catch((e: any) => {
```

Comment:
Why are we swallowing this error? I think this will result in `mintCost` being `undefined` if it throws an error?

Reply:
I was trying to find the revert error. It was a debug statement that should be removed.